modules/olm-mirroring-catalog.adoc

```diff
@@ -137,6 +138,7 @@ $ oc adm catalog mirror \
     <mirror_registry>:<port>/<namespace> \//<2>
     [-a ${REG_CREDS}] \
     [--insecure]
+    [--index-filter-by-os='<platform>/<arch>']
```

Comment:
Because this is now the new final line of the multi-line code example, the line before it (`[--insecure]` on line 140) should now gain a `\` at the end, so that it can be copy-pasted into a terminal correctly. So the full code block would look like:

```
$ oc adm catalog mirror \
    file://local/index/<repo>/<index_image>:<tag> \//<1>
    <mirror_registry>:<port>/<namespace> \//<2>
    [-a ${REG_CREDS}] \
    [--insecure] \
    [--index-filter-by-os='<platform>/<arch>']
```

See the "For long lines of code that you want to break up among multiple lines..." example in https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output